app/src/main/java/com/example/weightcontrolapp/ui/history/HistoryFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.weightcontrolapp.ui.history
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.weightcontrolapp.R
+
+class HistoryFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        super.onCreateView(inflater, container, savedInstanceState)
+
+        return inflater.inflate(R.layout.history_fragment, container, false)
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+    }
+
+    override fun onPause() {
+        super.onPause()
+    }
```

Comment:
It's the same as [this](https://github.com/kenta7777/WeightControlApp/pull/6/files#r502883124).

Reply:
done.